src/datasources/accounts/accounts.datasource.spec.ts

```diff
@@ -88,20 +91,126 @@ describe('AccountsDatasource tests', () => {
       );
     });
 
-    it('throws when an account with the same address already exists', async () => {
+    it('creates an account successfully if the clientIp is not a valid IP', async () => {
```

Comment:
```suggestion
    it('creates an account successfully if the clientIp is a valid IP', async () => {
```

Reply:
I guess we will see how this goes through time, I don't think user can obfuscate their IPs, but if they do, I think we can start allowing account creation but keep an eye on logging to see if this (a malformed IP reach the service) happens often. So I'd leave it as it is for now, and maybe reconsider it based on the logs.